plugins/keycloak-backend/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@
 You can run a development setup using the following command:
 
 ```console
-yarn start
+yarn run start --filter=@janus-idp/backstage-plugin-keycloak-backend
```

Comment:
@tumido any concerns with this?

Reply:
I think I have concerns, yes:

It doesn't make any difference in dev setup. Caching is disabled for `start` task. So the merit of this change is NOOP in the end. Hence why doing it?

This change requires you to run the command from within the repository root *only*. You can't execute this from within the `plugins/*`. The `yarn start` must be executed from within `plugins/*` only and `yarn workspace @janus-idp/backstage-plugin-* start` can be executed from both, repo root and package root. See below.



```shell
# for simplicity let's use topology plugin

$ cd plugins/topology

$ yarn run start --filter=@janus-idp/backstage-plugin-topology        
yarn run v1.22.19
$ backstage-cli package start --filter=@janus-idp/backstage-plugin-topology
error: unknown option '--filter=@janus-idp/backstage-plugin-topology'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

$ yarn start                                                  
yarn run v1.22.19
$ backstage-cli package start
Loaded config from app-config.yaml
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/, http://[::1]:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/tcoufal/Programming/janus-idp/backstage-plugins/plugins/topology/public' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
<i> [webpack-dev-middleware] wait until bundle finished: /
webpack compiled successfully
^C<i> [webpack-dev-server] Gracefully shutting down. To force exit, press ^C again. Please wait...

$ yarn workspace @janus-idp/backstage-plugin-topology start
yarn workspace v1.22.19
yarn run v1.22.19
$ backstage-cli package start
Loaded config from app-config.yaml
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/, http://[::1]:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/tcoufal/Programming/janus-idp/backstage-plugins/plugins/topology/public' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
<i> [webpack-dev-middleware] wait until bundle finished: /
webpack compiled successfully
^C<i> [webpack-dev-server] Gracefully shutting down. To force exit, press ^C again. Please wait...

$ cd ../..

$ yarn workspace @janus-idp/backstage-plugin-topology start
yarn workspace v1.22.19
yarn run v1.22.19
$ backstage-cli package start
Loaded config from app-config.yaml
<i> [webpack-dev-server] Project is running at:
<i> [webpack-dev-server] Loopback: http://localhost:3000/, http://[::1]:3000/
<i> [webpack-dev-server] Content not from webpack is served from '/Users/tcoufal/Programming/janus-idp/backstage-plugins/plugins/topology/public' directory
<i> [webpack-dev-server] 404s will fallback to '/index.html'
<i> [webpack-dev-middleware] wait until bundle finished: /
webpack compiled successfully
^C<i> [webpack-dev-server] Gracefully shutting down. To force exit, press ^C again. Please wait...

```

Therefore I would like to ask you @Zaperex to change this command to `yarn workspace @janus-idp/backstage-plugin-* start` for every package. It is the least confusing option.

Reply:
@Zaperex Oh, right! I understand now 🙂 Then I think the question is: do we need to run `generate` before every `start`? Or is this something you run only once and it's all set for you?

In case you need to re-`generate` before every `start`, it makes sense to involve turbo. Otherwise we don't need it and the update to `CONTRIBUTING.md` you suggest makes more sense to me. 🙂